### PR TITLE
Switch to using CSS

### PR DESCRIPTION
**Pull request by lramos15**
My piece of https://github.com/microsoft/vscode/issues/165169

### CODE REVIEW

File: src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css [hunk 1]
```diff
@@ -820,3 +820,141 @@
...
+	background: var(--vscode-welcomePage-tileBackground);
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer .gettingStartedSlideCategories .getting-started-category {
+	filter: drop-shadow(2px 2px 2px var(--vscode-welcomePage-tileShadow));
+}
+
+.monaco-workbench .part.editor > .content .gettingStartedContainer button:hover {
+	background: var(--vscode-welcomePage-tileHoverBackground);
+	outline-color: var(--vscode-contrastActiveBorder);
```
Comment (lramos15) [inline src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:863]: This is causing me trouble, not sure what the solution is. This color token is null in the dark, and light theme. Meaning that normally in the `registerThemeingParticipant` it wouldn't be set at all, but here it seems to default to white causing a difference in styling. I tried playing around with it for awhile, but don't know a good fix to keep the current style we have now. Should we define this for the other theme types too? That feels unnecessary

Comment (jrieken) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:863]: You can use `var(--name, <fallback>`) like `outline-color: var(--vscode-contrastActiveBorder, inherit)` or so. That is still extra styling but not visible

Comment (lramos15) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:863]: Hmm. That didn't change anything. Is there a good way to debug CSS variables. Maybe I need a set of fresh eyes as I can't find a difference besides that line

Comment (jrieken) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css:863]: 🤔 I haven't tried it myself but that's how it should work... https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values.

Comment (lramos15) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css]: The fallback works, it's just there's no way to act like the rule just isn't there if the variable is not defined. I've fallen back to the focus border color which is what would normally be applied anyways if the value isn't set. Just something to be aware of.

**Review by jrieken — APPROVED** [review]

**Pull request merged**